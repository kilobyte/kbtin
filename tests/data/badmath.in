#math x ()
#math x 1/
#math x 1/0
#math x ))
