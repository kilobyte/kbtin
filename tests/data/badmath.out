#Invalid expression to evaluate in {()}
#Invalid expression to evaluate in {1/}
#Error: Division by zero.
#Unmatched parentheses error in {))}.
